mappings/net/minecraft/client/render/debug/GameEventDebugRenderer.mapping

```diff
@@ -12,8 +12,18 @@ CLASS net/minecraft/class_5739 net/minecraft/client/render/debug/GameEventDebugR
 	METHOD method_33088 addListener (Lnet/minecraft/class_5716;I)V
 		ARG 1 positionSource
 		ARG 2 range
+	METHOD method_33089 drawBoxIfCameraReady (Lnet/minecraft/class_238;FFFF)V
```

Comment:
What is camera ready? Like this is not hidden by the frustum?

Reply:
I named it like that because the actual code is inside an `if (camera.isReady())`